### PR TITLE
Add run durations to UI.

### PR DESCRIPTION
**Pull request by JuhaS**
@zbal added this last night to have durations of tasks visible in UI, looks ok?

image

### CODE REVIEW

File: app/client/components/Task/Task.js [hunk 1]
```diff
@@ -269,6 +269,19 @@ export default class Task extends Component {
       return <div className={`status ${status}`} key={run.id}><span>{run.status} &middot; { that.relativeTime(run.start_time)}</span></div>
     })
   }
+  calculateRunDuration(run) {
+    var duration;
+    if (run['start_time'] && run['finish_time']){
+      try {
+        duration = (moment(run['finish_time']) - moment(run['start_time']))/1000
+        duration = (duration + 1).toFixed(0) // round to integer
```
Comment (makara) [inline app/client/components/Task/Task.js]: `moment` can display it a bit nicer: http://momentjs.com/docs/#/durations/humanize/

**Comment by zbal** [comment]
I like it

File: app/client/components/Task/Task.js (hunk 1 above)
Comment (zbal) [reply app/client/components/Task/Task.js]: I guess we already use the humanized version after, here - we try to be precise with the exat duration?

Comment (JuhaS) [reply app/client/components/Task/Task.js]: I just wanted to keep it compact to not overflow the area. But yeah, we'll at least need to show minutes if there are very long tasks.

**Comment by haishanh** [comment]
any reason why this is not merged?

**Comment by JuhaS** [comment]
I think this is ready to be merged, I'll do that next.

**Pull request merged**